File: hw1-kyle-pish/p2/p2.cpp
/*
Kyle Pish
CS355
Assignment 1 p2.cpp
*/

#include <iostream>
#include <math.h>
#include <iomanip>
int main() {

    //variable declarations
    double diameter;
    double price;
    double cubicInches;
    double pricePerCubicInch;
    double radius;
    
   
    //promt user input
    std::cout << "Enter the diameter of the canteloupe in inches: " << std::endl;
    std::cin >> diameter;

    std::cout << "Enter the price of the canteloupe in dollars: " << std::endl;
    std::cin >> price;

    //calculations
    radius = diameter / 2;
    cubicInches = (4 * M_PI * pow(radius, 3))/3;
    pricePerCubicInch = price / cubicInches;

    //print the final varlue rounded to 2 decimal places
    std::cout << std::setprecision(2) << pricePerCubicInch << std::endl;
    



}

File: hw1-kyle-pish/p2/p2.py
'''
Kyle Pish
CS355
Assignment 1 p2.py
'''

#import math for access to math.pi
import math

#variable declaration and prompt user for input
diameter = input("Enter the diameter of the canteloupe in inches: \n")
diameter = float(diameter)
price = input("Enter the price of the canteloupe in dollars: \n")
price = float(price)

#calculations
CubicInches = ((4/3) * math.pi * ((diameter/2)**3))
#calculation of final value ronded to 2 decimal points
PricePerCubicInch = round (price / CubicInches, 2)

#output final value
print (PricePerCubicInch)